docs/src/DeveloperDocumentation/styleguide.md

```diff
@@ -57,7 +57,12 @@ Here is a summary of the naming convention followed in OSCAR:
   also being consistent.
   For compatibility with standard Julia, while staying consistent internally,
   we also provide aliases (using `AbstractAlgebra.@alias`) for various standard
-  Julia functions, e.g. `is_one` as alias for `isone`
+  Julia functions, e.g. `is_one` as alias for `isone`.
+- A function returning the number of some things should be named `number_of_things`.
+  For some very common things, like the number of generators, we additionally
+  provide a shorter alias, e.g. `ngens` for `number_of_generators`. These aliases should be
+  short (< 15 chararacters) and without underscores, and one should only use
+  them inside of functions, not in the user interface and documentation.
```

Comment:
I think this is not correct anymore, but I am not sure what the correct new description would be. Perhaps @lkastner @benlorenz @micjoswig could propose something?

In particular I am not quite sure myself. So the short aliases now start with `n_`, but we keep `number_of_`? Or do we plan to phase out `number_of_` in favor of `n_` (I'd be fine with that, too, just need to know what the plan is).

Thanks!